tutorials/circuits/01_circuit_basics.ipynb

```diff
@@ -8118,9 +8118,9 @@
     "from qiskit import transpile \n",
```

Comment:
<p>Thanks for your contribution! I think the preferred syntax here is similar to the previous version. You can import AerSimulator</p><p> </p><pre class="ql-syntax" spellcheck="false"># Use AerSimulator
from qiskit.providers.aer import AerSimulator

backend = AerSimulator()
</pre><p> </p><p>Can you please make the suggested changes? Thank you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Thank you for the suggestion!
I have made the changes as proposed and would appreciate your confirmation.